### PR TITLE
testing

### PR DESCRIPTION
**Pull request by duglin**
Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

**Comment by duglin** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -1,8 +1,5 @@
 [![Build Status](https://travis-ci.org/openservicebrokerapi/servicebroker.svg?branch=master)](https://travis-ci.org/openservicebrokerapi/servicebroker "Travis")
 
-
```
Comment (duglin) [inline README.md]: LGTM

Comment (duglin) [reply README.md]: qweqwe

**Comment by n3wscott** [comment]
LGTM

**Comment by duglin** [comment]
LGTM

**Comment by duglin** [comment]
NOT LGTM

**Comment by duglin** [comment]
LGTM

**Comment by n3wscott** [comment]
NOT LGTM

**Comment by duglin** [comment]
LGTM

**Comment by duglin** [comment]
LGTM

**Comment by duglin** [comment]
merging just to remove testing stuff from README

**Pull request merged**